server/src/main/java/org/elasticsearch/common/geo/SphericalMercatorUtils.java

```diff
@@ -1,18 +1,26 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
```

Comment:
nit: same here, see the two copyright notices here. There are a few more cases in this PR so I'm leaving it at that in case this is done on purpose. Sorry for the noise in this case.